woocommerce/woocommerce-ios
# Introduce `Quarter to Date` and `Last Quarter` ranges

Closes #8146

Why
==========
The final part of issue #8146 is to deliver the `Yesterday`, `Last Week`, `Last Month`, `Last Year`, `Last Quarter` and `Quarter to Date` ranges. This one aims to the `Last Quarter`, and `Quarter to Date` ranges as the final batch of ranges.

How
==========
This PR introduces the `Last Quarter`, and `Quarter to Date` ranges, alongside the respective data structs and unit tests, updating the SelectionType and adding the three options to the Date range picker view.

Screen Capture
==========
https://user-images.githubusercontent.com/5920403/206017713-b06a0afb-4f83-4bfc-9adf-47a9a78af943.mp4

How to Test
==========
1. Open the Analytics Hub
2. Hit the Date range selection view
3. Verify that the `Last Quarter`, and `Quarter to Date` options are now available. 
4. Verify that the `Quarter to Date` now compares the current quarter to the previous one.
5. Verify that the `Last Quarter` now compares the previous full quarter with the full quarter before it.

Update release notes:
